packages/puppeteer-core/src/common/ElementHandle.ts

```diff
@@ -20,21 +20,100 @@ import {ExecutionContext} from './ExecutionContext.js';
 import {Frame} from './Frame.js';
 import {FrameManager} from './FrameManager.js';
 import {WaitForSelectorOptions} from './IsolatedWorld.js';
-import {
-  BoundingBox,
-  BoxModel,
-  ClickOptions,
-  JSHandle,
-  Offset,
-  Point,
-  PressOptions,
-} from './JSHandle.js';
+import {JSHandle} from './JSHandle.js';
 import {Page, ScreenshotOptions} from '../api/Page.js';
 import {getQueryHandlerAndSelector} from './QueryHandler.js';
 import {ElementFor, EvaluateFunc, HandleFor, NodeFor} from './types.js';
 import {KeyInput} from './USKeyboardLayout.js';
 import {debugError, isString} from './util.js';
 import {CDPPage} from './Page.js';
+import {MouseButton} from './Input.js';
+
+/**
+ * @public
+ */
+export interface BoxModel {
```

Comment:
Moved the Types to `ElementHandle` as they were unused in `JSHandle`.

Reply:
I think we can move these to the API file. WDYT?